src/components/policy/policy_external/src/update_status_manager.cc

```diff
@@ -86,6 +86,11 @@ void UpdateStatusManager::OnUpdateSentOut(uint32_t update_timeout) {
   ProcessEvent(kOnUpdateSentOut);
 }
 
+void UpdateStatusManager::OnUpdatePostponed() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  ProcessEvent(kPostponedUpdate);
```

Comment:
@ydementieiev consider using of `SetPostponedStatus` - maybe you can use it instead of `kPostponedUpdate`

Reply:
@AKalinich-Luxoft  I tried using `SetPostponedStatus` but always find a new regression after.  Now I renamed `OnUpdatePostponed` to name which more describe what this method do, look please